### ISSUE TITLE
nuxt module: nitro changed location of openapi.json

### ISSUE THREAD

**Issue by mirsella** [scalar/scalar#4204]
### What happens?

hello !

when going to /docs:
404
[GET] "/_nitro/openapi.json": 404 Page not found: /_nitro/openapi.json

because its now in /_openapi.json.
another issue as or even more important, nitro doesn't generate openapi.json in production, only in dev mode

thanks

### What did you expect to happen?

work

**Comment by mirsella** [comment]
hm this issue might be useless as we can use
```
  scalar: {
    spec: {
      url: 'https://cdn.jsdelivr.net/npm/@scalar/galaxy/dist/latest.yaml',
    },
  },
  ```
which i just learned overwrite the default

**Comment by hanspagel** [comment]
Thanks for the report! We should use the new default.

**Comment by mirsella** [comment]
yes ! but the most important thing is that by default (with the instruction of the nuxt module) it will not work with production nuxt app :)

**Comment by mirsella** [comment]
oh and the url in spec to change the spc url is absolute. someway of using a relative path would be awesome. for now i have localhost, then ill have to change it when pushing to prod, etc

**Comment by luc122c** [comment]
I've opened a PR to fix this. As a workaround, you can revert the endpoint to the old location in your config:


```ts
// nuxt.config.ts

nitro: {
	experimental: { 
		openAPI: true
 	},
	openAPI: {
		route: '/_nitro/openapi.json'
	}
}
```
